operator/api/v1alpha1/keptnappversion_types.go

```diff
@@ -29,8 +29,9 @@ import (
 
 // KeptnAppVersionSpec defines the desired state of KeptnAppVersion
 type KeptnAppVersionSpec struct {
-	KeptnAppSpec `json:",inline"`
-	AppName      string `json:"appName"`
+	KeptnAppSpec    `json:",inline"`
+	AppName         string `json:"appName"`
+	PreviousVersion string `json:"PreviousVersion,omitempty"`
```

Comment:
let's start the json tag with a lower case :)

Reply:
👍